### ISSUE TITLE
update require/import section of javascript.md

### ISSUE THREAD

**Issue by kevinbarabash** [Khan/style-guides#57]
- remove sorting of require/import statements
- blocks continues to be the same with the addition of a new block, flow type imports which should come last
- prefer ES6 `import` syntax over `require`
- destructuring should happen on the same line as the import, e.g. `import React, {Component} from 'react'`

**Comment by jlfwong** [comment]
:o did someone update kake to compile ES6 syntax?

**Comment by xymostech** [comment]
@jlfwong I believe it always did, but the require-finding regexes got confused. :) @jeresig fixed that!

**Comment by jlfwong** [comment]
Yeah, that's what I meant :) Glad to hear the artificial restriction to the code I wrote long ago isn't holding anyone back :D
